src/components/transport_manager/src/iap2_emulation/iap2_transport_adapter.cc

```diff
@@ -58,17 +73,86 @@ IAP2USBEmulationTransportAdapter::IAP2USBEmulationTransportAdapter(
     const uint16_t port,
     resumption::LastState& last_state,
     const TransportManagerSettings& settings)
-    : TcpTransportAdapter(port, last_state, settings) {}
+    : TcpTransportAdapter(port, last_state, settings), out_(0) {
+  auto delegate = new IAPSignalHandlerDelegate(this);
+  signal_handler_ =
+      threads::CreateThread("iAP signal handler", delegate);
+  signal_handler_->start();
+  LOG4CXX_DEBUG(logger_, "Out signals channel creation result: "
+                << mkfifo(out_signals_channel, mode));
```

Comment:
@dev-gh do not move logical code inside debug output. Will be missed in case of LOGS=OFF

Reply:
@LuxoftAKutsan 
Done with 0f8e14b